src/getCoinCombination.test.js

```diff
@@ -1,10 +1,41 @@
 'use strict';
 
-describe('getCoinCombination', () => {
-  const { getCoinCombination } = require('./getCoinCombination');
+describe('getHumanAge', () => {
```

Comment:
The test suite is named 'getHumanAge', which does not match the task description for testing 'getCoinCombination'. Consider renaming this to reflect the correct function.